packages/interface/tsconfig.json

```diff
@@ -0,0 +1,21 @@
+{
+  "compilerOptions": {
+    "target": "es2018",
```

Comment:
Set target to 2017 until we upgrade all of our targets.

Reply:
Done.